test/lib-test/ProblemList.txt

```diff
@@ -37,3 +37,4 @@
 # More than one label is allowed but must be on the same line.
 #
 #############################################################################
+jdk/test/lib/TestMutuallyExclusivePlatformPredicates.java 8286191 generic-all
```

Comment:
This change shouldn't be here. I think you need to update
your repo to sync with your earlier fix.

Reply:
Updated, after the merge only the langtools ProblemList.txt is modified